src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -348,4 +349,22 @@ void setParent(hudson.tasks.junit.TestResult parent) {
     private static final long serialVersionUID = 1L;
 
     private static final Pattern SUREFIRE_FILENAME = Pattern.compile("TEST-(.+)\\.xml");
+
+    /**
+     * Merges another SuiteResult into this one.
+     * 
+     * @param sr
```

Comment:
If you define the parameter in Javadoc, please add its description. Otherwise Javadoc linters will grumble